### ISSUE TITLE
How to run sofa in Docker Image, I can not find any instructions to follow up

### ISSUE THREAD

**Issue by caimingxue** [sofa-framework/sofa#4230]
Hi, I have installed the SOFA docker Image, but I have no idea how to run sofa in Docker!

**Comment by bakpaul** [comment]
Hi ! 
What image are you exactly referring to ? Can you provide the docker-hub adress ?

**Comment by caimingxue** [comment]
Thanks for your reply. I have installed the following docker images according to https://hub.docker.com/r/sofaframework/sofabuilder_ubuntu/tags.
sofaframework/sofa_nightly_ubuntu   master-standard   1feb9eeaf1ab   12 hours ago   3.06GB
sofaframework/sofabuilder_ubuntu    latest            fe182d39bb54   5 days ago     8.65GB
sofaframework/sofabuilder_ubuntu    v23.06            ca308031fc67   7 weeks ago    8.62GB
My question is I do not how to runSofa in the docker container. Do you have any suggestions how to use the sofa docker images?

**Comment by bakpaul** [comment]
Yes, this image is supposed to be used for compilation purpose. It doesn't contain sofa itself but all the dependency required to compile sofa and use it. So you can use it to compile sofa yourself or configure your own environment to do so.

**Comment by caimingxue** [comment]
That is to say, I need to download the source of the sofa in the docker container (/home), then compile the sofa according to the regular instructions based on dependencies in the docker

**Comment by bakpaul** [comment]
Yes exactly. Normally runing a simple cmake with the desired flags to activate or deactivate some plugins will be enough. All dependency are met for most of the plugins.

**Comment by naixuedecha6** [comment]
can make a tutorial? Thanks
